packages/compose/src/hooks/use-copy-to-clipboard/index.js

```diff
@@ -20,7 +20,9 @@ import useRefEffect from '../use-ref-effect';
  */
 function useUpdatedRef( value ) {
```

Comment:
I think the need for this hook can probably be removed entirely and replaced with `useEvent`

Reply:
That would work for `onSuccess`, but not with `text` since it can be `string|callback` and the `useEvent` always expected callback.

Reply:
It would work for both because onSuccess will just be using the latest "text" there's no need for hook around text.

Reply:
Actually you maybe right here, text seems to be used outside of the event handler, (within the refEffect).